crates/ruff_linter/src/rules/pygrep_hooks/rules/invalid_mock_access.rs

```diff
@@ -89,6 +96,13 @@ pub(crate) fn non_existent_mock_method(checker: &Checker, test: &Expr) {
             | "called_with"
             | "has_calls"
             | "not_called"
+            | "awaited"
```

Comment:
N.B., the sync equivalent `called` is not in this list. That is because (sync) `Mock`s have a boolean attribute `called`. However, `AsyncMock`s do not have an attribute `awaited`. Therefore, `assert mock.awaited` is likely a mistake and was meant to be `mock.assert_awaited()`.